python/ray/serve/controller.py

```diff
@@ -257,9 +311,20 @@ async def deploy(self,
                 replica_config=replica_config,
                 deployer_job_id=deployer_job_id,
                 start_time_ms=int(time.time() * 1000))
+            # TODO(architkulkarni): When a deployment is redeployed, even if
+            # the only change was num_replicas, the start_time_ms is refreshed.
+            # This is probably not the desired behavior for an autoscaling
+            # deployment, which redeploys very often to change num_replicas.
```

Comment:
please file issues to track these TODOs -- can you make an autoscaling milestone?